src/infra/core/Cargo.toml

```diff
@@ -151,6 +151,9 @@ libc = "0.2" # For getting uid:gid
 database-common = { workspace = true }
 kamu-accounts-inmem = { workspace = true }
 kamu-accounts-services = { workspace = true }
+kamu-auth-rebac = { workspace = true, default-features = false }
+kamu-auth-rebac-inmem = { workspace = true }
+kamu-auth-rebac-services = { workspace = true }
```

Comment:
These dependencies are unexpected in core tests too.

Reply:
Removed -- Done